src/python-lxc/lxc.c

```diff
@@ -25,14 +25,91 @@
 #include <Python.h>
 #include "structmember.h"
 #include <lxc/lxccontainer.h>
-#include "lxc/utils.h"
-#include "lxc/namespace.h"
-#include "lxc/confile.h"
 #include <stdio.h>
 #include <sys/wait.h>
+#include <sched.h>
+
+/*
+ * CLONE_* definitions copied from lxc/namespace.h
+ */
+#ifndef CLONE_FS
+#  define CLONE_FS                0x00000200
+#endif
+#ifndef CLONE_NEWNS
+#  define CLONE_NEWNS             0x00020000
+#endif
+#ifndef CLONE_NEWCGROUP
+#  define CLONE_NEWCGROUP         0x02000000
+#endif
+#ifndef CLONE_NEWUTS
+#  define CLONE_NEWUTS            0x04000000
+#endif
+#ifndef CLONE_NEWIPC
+#  define CLONE_NEWIPC            0x08000000
+#endif
+#ifndef CLONE_NEWUSER
+#  define CLONE_NEWUSER           0x10000000
+#endif
+#ifndef CLONE_NEWPID
+#  define CLONE_NEWPID            0x20000000
+#endif
+#ifndef CLONE_NEWNET
+#  define CLONE_NEWNET            0x40000000
+#endif
+
+/* From sys/personality.h */
+#define PER_LINUX 0x0000
+#define PER_LINUX32 0x0008
```

Comment:
At least on Arch, sys/personality.h is a part of glibc package, and it is not shipped by any other libc implementation. So this will not compile with any other libc (not sure if it is actually a problem).
PER_LINUX* were added to glibc in 2002: https://sourceware.org/git/?p=glibc.git;a=commit;f=sysdeps/unix/sysv/linux/sys/personality.h;h=61ae62f27037d4c38acca2533a06dcd17fbc2ca2 so they should be present in all supported distros.

But for CLONE_* definitions everything is different:
$ grep -R 'CLONE_NEWCGROUP' /usr/include/
prints nothing. I don't have CLONE_NEWCGROUP defined in any system header, on a up-to-date Arch Linux installation. Usually it means these definitions aren't available in any released glibc version.